homeassistant/components/warmup4ie/__init__.py

```diff
@@ -0,0 +1 @@
+"""The warmup4ie component."""
```

Comment:
Please rename this to `warmup`, not `warmup4ie`.

Reply:
Okay. Done.